### PR TITLE
Add --repeat option

### PR DESCRIPTION
**Pull request by phihag**
To identify flaky tests, it is very helpful to be able to re-run them multiple times.
Add a `--repeat` option that does exactly that.

### CODE REVIEW

File: output.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ function status(config, state) {
     let status_str;
     for (let running_show = running.length;running_show >= 0;running_show--) {
         const running_str = (
-            running.slice(0, running_show).map(({tc}) => tc.name).join(' ')
+            running.slice(0, running_show).map(task => task.name).join(' ')
```
Comment (marvinhagemeister) [inline output.js:43]: I'm not sure I fully understand this line. Looking at the other changes in the PR it seems like `task.tc.name` holds the proper name like here: https://github.com/boxine/pentf/pull/97/files#diff-2375abc0904a66fd5a4a879c9b129fadR25

Did we change the shape of the `task` object?

File: render.js [hunk 2]
```diff
@@ -156,50 +183,83 @@ function screenshots_html(result) {
...
+        .join(' '));
+}
+
+function _calcDuration(taskResults) {
+    if (taskResults.length === 1) {
+        return format_duration(taskResults[0].duration);
+    }
+
+    const max = Math.max(... taskResults.map(tr => tr.duration));
+    const min = Math.min(... taskResults.map(tr => tr.duration));
```
Comment (marvinhagemeister) [inline render.js:216]: Minor nit: We could skip the second iteration by storing the duration array in a variable.

Comment (phihag) [reply render.js]: Excellent point! Fixed. If only JavaScript had [generator expressions](https://docs.python.org/dev/reference/expressions.html#grammar-token-generator-expression) ;)

File: output.js (hunk 1 above)
Comment (phihag) [reply output.js:43]: No, the task object has stayed the same. However, here we used to output the test case (`tc`) name. Now, we output the task name. By default, test case and task name are the same.
But with `--repeat` (or future retesting), the test name is unchanged (something like `selftest_browser`), but the task name will include the repetition, e.g. `selftest_browser[4]`

**Pull request merged**